src/utils/getFilteredTodos.ts

```diff
@@ -0,0 +1,39 @@
+import { Todo } from '../types/Todo';
+import { Filters } from '../types/Filters';
+
+type Params = {
+  query: string;
+  selectedFilter: Filters;
+};
+
+export const getFilteredTodos = (
+  todos: Todo[],
+  { query, selectedFilter }: Params,
+) => {
+  let filteredTodos = [...todos];
+
+  filteredTodos = filteredTodos.filter(todo => {
+    switch (selectedFilter) {
+      case Filters.ACTIVE:
+        return !todo.completed;
+
+      case Filters.COMPLETED:
+        return todo.completed;
+
+      default:
+        return todo;
```

Comment:
In the default case of the switch statement, you're returning the entire todo object, which will always be truthy. This means that if the selected filter is not 'ACTIVE' or 'COMPLETED', all todos will be included in the `filteredTodos` array, regardless of their completed status. If this is the intended behavior, it would be clearer to return `true` in the default case to explicitly indicate that all todos should be included.